src/EventTeachingService.cpp

```diff
@@ -150,7 +152,7 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
 
             //DEBUG_SERIAL << F("> sending ENRSP reply for event index = ") << i << endl;
             controller->sendMessage(msg);
-            delay(10);
+ //           delay(10);
```

Comment:
If it works without this delay, please remove this line. No need for such commented code.

Reply:
Done